initrd/bin/gui-init

```diff
@@ -438,6 +438,7 @@ show_options_menu()
     'C' ' Reencrypt LUKS container -->' \
     'P' ' Change LUKS Disk Recovery Key passphrase ->' \
     'R' ' Check/Update file hashes on root disk -->' \
+    'h' ' Change system time' \
```

Comment:
@alexgithublab 

- Why under gui-init option menu?
  - if under Options menu? Why so low in this menu?
- Why not proposed under TOTP mismatch prompt as replacement for current workflow?